backward_paper/evolve_backward.py

```diff
@@ -55,27 +57,35 @@ def evolve_backward(self, pdf_name, q_high=1.65, q_low=1.5):
 
         self.run([theory_updates], [operator_updates], [pdf_name])
 
-    def evolve_above_below_thr(self, pdf_name, heavy_quark="c", epsilon=0.01):
+    def evolve_above_below_thr(
+        self, pdf_name, q_high=1.65, heavy_quark="c", epsilon=0.01
+    ):
         """
         Comapare above and below the heavy quark threshold
 
         Parameters
         ----------
             pdf_name: str
                 PDF name
+            q_high: float
+                initial Q scale
             heavy_quark: str
                 heavy quark name
             epsilon: float
                 distance from threshold
         """
-        self.fig_name = f"compare_thr_{pdf_name}"
+        self.fig_name = f"compare_thr_{heavy_quark}_{pdf_name}"
+        self.plot_pdfs = [-pid_dict[heavy_quark], pid_dict[heavy_quark]]
+
         operator_updates = self.base_operator
         thr_scale = (
             self.base_theory[f"m{heavy_quark}"] * self.base_theory[f"k{heavy_quark}Thr"]
         )
         operator_updates["Q2grid"] = np.power(
             [thr_scale + epsilon, thr_scale - epsilon], 2
         )
+        theory_updates = self.base_theory
+        theory_updates["Q0"] = q_high
         self.run([self.base_theory], [operator_updates], [pdf_name])
```

Comment:
why do you cook theory_updates and then don't use it? I guess that's a typo ;-)
however, if you want you can even overwrite `run` to mixin the base_theory

Reply:
It's a typo